entrypoint.sh

```diff
@@ -11,20 +17,20 @@ if [[ -d "${CASE_PATH}" ]]; then
         for entry in "${CASE_PATH}"/*
         do
             echo "Validating file ${entry} found within ${CASE_PATH}"
-            case_validate "${entry}" --built-version "${CASE_VERSION}"
+            eval "${cmd} ${entry}"
```

Comment:
Is `eval` strictly necessary?  As a matter of development safety, I encourage avoiding `eval` whenever possible because it becomes much harder to statically analyze code safety.

Reply:
It makes it easier to build the command rather than having to control multiple, duplicative, if/then/else blocks.

Reply:
I won't halt progress on `eval`.  I suspect there is a safer mechanism using arrays (e.g. `"$@"`'s quote expansion).

However, looking at this closer, it looks like this quoting pattern might not preserve quotes for `${entry}`, so that may be a separate looming bug.  Can you test this on a file with a whitespace in its name, `foo 1.txt`?

Reply:
Got rid of the `eval` calls and quoted the filenames. I also added a test file that has a space in the filename (and also tested the directory filter)

Reply:
Looks good to me!